socrates/lib/registration/views/participate.jade

```diff
@@ -15,8 +15,13 @@ block content
     if(registrationTuple.duration === 'waitinglist')
       h1 Waitinglist Registration<br>
     else
-      h1 Registration<br>
-      small Personal Information
+      h1 Registration #{' '}
+        small Personal Information
+      
+    p.
+      You have #{expiresIn} minutes to complete your registration. The reservation will expire 
```

Comment:
Eher vielleicht "to submit this form" order "to fill in this form and send it to us" oder so. Ich finde es unklar, was "complete your registration" bedeuten soll...